pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -145,6 +146,8 @@ public Object handle(Request request, Response response, UserRegistrationPayload
                 // if the invitation is revoked or there's already an account for the user, error out.
                 if (invitation.isVoid()) {
                     throw new DDPException("Invalid invitation " + invitationGuid + " for user " + auth0UserId);
+                } else if (invitation.isAccepted()) {
```

Comment:
I think we should also check if invitation has already been accepted?

Reply:
Yep!  Thanks.